packages/uhk-web/src/app/components/agent/about/about.component.ts

```diff
@@ -11,7 +12,15 @@ import { getVersions } from '../../../util';
         'class': 'container-fluid'
     }
 })
-export class AboutComponent {
+export class AboutComponent implements OnInit {
```

Comment:
Maybe I overengineering the app, but I like to separate the responsibility of the elements.
In my solution, we create a reducer and actions to load and store the contributors and we handle the loading and error state too.

It is up to you to follow these suggestions or not, but I believe automatic testing will be easier for the app.